tests/test_admin.py

```diff
@@ -51,3 +55,120 @@ def test_language_filter(self):
+            response = self.client.get(base_url)
+
+        response_content_decoded = response.content.decode()
+
+        # Check Column Headings
+        self.assertInHTML(
+            'Category',
+            response_content_decoded,
+        )
+
+        # Check Alias content row values
+        self.assertIn(
+            category.name,
+            response_content_decoded
+        )
+        self.assertIn(
+            expected_en_content.name,
+            response_content_decoded,
+        )
+
+    @skipUnless(is_versioning_enabled(), 'Test only relevant for versioning')
+    def test_alias_content_manager_rendering_with_versioning_actions(self):
+        category = Category.objects.create(name='Language Filter Category')
+        alias = AliasModel.objects.create(
+            category=category,
+            position=0,
+        )
+        expected_en_content = AliasContent.objects.create(
+            alias=alias,
+            name="EN Alias Content",
+            language="en",
+        )
+        # If versioning is enabled be sure to create a version
+        if is_versioning_enabled():
+            from djangocms_versioning.models import Version
+
+            Version.objects.create(content=expected_en_content, created_by=self.superuser)
+
+        base_url = self.get_admin_url(AliasContent, "changelist")
+
+        with self.login_user_context(self.superuser):
+            # en is the default language configured for the site
+            response = self.client.get(base_url)
+
+        response_content_decoded = response.content.decode()
+
+        # Check Column Headings
+        self.assertInHTML(
+            'Category',
+            response_content_decoded,
+        )
+        self.assertInHTML(
+            'Author',
+            response_content_decoded,
+        )
+        self.assertInHTML(
+            'Modified',
+            response_content_decoded,
+        )
+        self.assertInHTML(
+            'State',
+            response_content_decoded,
+        )
+        self.assertInHTML(
+            'Actions',
+            response_content_decoded,
+        )
+
+        # Check Alias content row values
+        self.assertIn(
+            category.name,
+            response_content_decoded
+        )
+        self.assertIn(
+            expected_en_content.name,
+            response_content_decoded,
+        )
+
+        latest_alias_content_version = expected_en_content.versions.all()[0]
+
+        self.assertInHTML(
+            f'<td class="field-get_author">{latest_alias_content_version.created_by.username}</td>',  # noqa: E501
+            response_content_decoded,
+        )
+        self.assertIn(
+            latest_alias_content_version.get_state_display(),
+            response_content_decoded,
+        )
+        self.assertIn(
+            localtime(
+                latest_alias_content_version.modified
+            ).strftime("%B %-d, %Y, %-I:%M %p").replace('AM', 'a.m.').replace('PM', 'p.m.'),
```

Comment:
Manipulating a date string like this should not be required i.e. with the replace. The admin will get it's string format from the settings, you should be able to import this from the settings loaded to use that: 

https://docs.djangoproject.com/en/3.2/ref/settings/#datetime-format

Reply:
Code updated based on comments